### PR TITLE
updated readme for typo in configure

### PR DESCRIPTION
**Pull request by PatrickAlphaC**
This addresses #3857

### CODE REVIEW

**Review by se3000 — APPROVED** [review]

**Review by se3000 — APPROVED** [review]

**Review by se3000 — APPROVED** [review]

**Review by se3000 — DISMISSED** [review]

**Comment by alexroan** [comment]
@se3000 is there a way we can retroactively sign old commits? Merging is blocked on this until we do so.

**Comment by se3000** [comment]
@alexroan rebasing or amending should sign the commit for you

File: README.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ Check out the [docs'](https://docs.chain.link/) pages on [Adapters](https://docs
 
 ## Configure
 
-You can configure your node's behavior by setting environment variables which can be, along with default values that get used if no corresponding environment variable is found. The latest information on configuration variables are available in [the docs](https://docs.chain.link/docs/configuration-variables).
```
Comment (sexylexy69) [inline README.md:96]: Help

File: README.md [hunk 2]
```diff
@@ -93,7 +93,7 @@ Check out the [docs'](https://docs.chain.link/) pages on [Adapters](https://docs
...
+You can configure your node's behavior by setting environment variables. All the environment variables can be found in the `ConfigSchema` struct of `schema.go`, or you can read the [official documentation](https://docs.chain.link/docs/configuration-variables) to learn the most up to date information on each of them. For every variable, default values get used if no corresponding environment variable is found.
```
Comment (sexylexy69) [inline README.md:96]: Asap

Comment (sexylexy69) [inline README.md:96]: Asap

**Pull request closed without merging**